packages/react/src/VideoPlayer/VideoPlayer.module.css

```
@@ -0,0 +1,405 @@
+}
+
+.VideoPlayer__rangeInput::-moz-range-track,
+.VideoPlayer__rangeInput::-webkit-slider-runnable-track {
+  background: transparent;
+  border: none;
+}
+
+.VideoPlayer__rangeProgress {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  overflow: hidden;
+  background: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+  border: 0;
+  border-radius: var(--brand-VideoPlayer-range-borderRadius);
+  width: 100%;
+  position: absolute;
+  height: var(--brand-VideoPlayer-range-height);
+  top: 0;
+}
+
+.VideoPlayer__rangeProgress::-webkit-progress-bar {
+  background-color: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+}
+
+.VideoPlayer__rangeProgress::-webkit-progress-value {
+  background-color: var(--base-color-scale-blue-5);
+}
+
+.VideoPlayer__rangeProgress::-moz-progress-bar {
+  background: var(--base-color-scale-blue-5);
+}
+
+.VideoPlayer__rangeProgress::-ms-fill-lower {
+  background-color: var(--base-color-scale-blue-5);
+  border: none;
+}
+
+.VideoPlayer__rangeProgress::-ms-fill-upper {
+  background-color: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+  border: none;
+}
+
+/* ---------------------------------------------------------- */
+/* 7. Tooltip                                                 */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer__tooltip {
+  position: absolute;
+  bottom: 100%;
+  /* left: 50%; */
+  z-index: 2;
+  width: 10px;
+  height: 10px;
+  margin-bottom: 16px;
+  pointer-events: none;
+  opacity: 0;
+  transform: translate3d(0%, 50%, 0);
+  transition: opacity 0.5s cubic-bezier(0.16, 1, 0.3, 1), transform 0.5s cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.VideoPlayer__tooltip::before {
+  position: absolute;
+  bottom: -4px;
+  left: 0;
+  width: 10px;
+  height: 10px;
+  content: '';
+  background: var(--base-color-scale-white-0);
+  border-radius: 2px;
+  transform: rotate(45deg);
+}
+
+.VideoPlayer__tooltipContent {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  z-index: -1;
+  background: white;
+  box-shadow: 0 4px 4px hsl(var(--base-color-scale-black-0-hsl) / 20%);
+  transform: translate3d(-50%, 0, 0);
+  padding: var(--brand-VideoPlayer-tooltip-padding);
+  border-radius: var(--brand-borderRadius-medium);
+}
+
+.VideoPlayer__tooltipText {
+  color: var(--brand-InlineLink-color-rest);
+}
+
+.VideoPlayer__controlTextColor {
+  color: var(--base-color-scale-white-0);
+}
+
+.VideoPlayer__shiftTooltipRight .VideoPlayer__tooltip {
+  transform: translateX(10px);
+}
+
+/* ---------------------------------------------------------- */
+/* 8. Captions                                                */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer__captions {
+  width: 90%;
+  margin: var(--brand-VideoPlayer-closedCaption-margin);
+  display: grid;
+  justify-content: center;
+  pointer-events: none;
+}
+
+.VideoPlayer__captions__empty {
+  display: none;
+}
+
+.VideoPlayer__captions__text {
+  color: white;
```

Can we use our tokens instead of named colors please?

```suggestion
  color: var(--base-color-scale-white-0);
```

---

This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5